.circleci/config.yml

```diff
@@ -501,6 +501,9 @@ jobs:
       - run:
           name: test:coverage
           command: yarn test:coverage
+      - run:
+          name: test:coverage:jest
+          command: yarn test:coverage:jest
```

Comment:
It would be nice to store these coverage reports somewhere :thinking: But `coverage` is taken. Maybe we can set the `coverageDir` to `jest-coverage` or something? Then we could persist it a couple lines down from here.